### DISCUSSION TITLE
Register now for GitHub Universe 2022

### DISCUSSION THREAD

**Discussion by tuves** [community/community#35799] [Announcements]
Join the global developer event for cloud, security, community, and AI. Grab your [tickets](https://githubuniverse.com/) and experience it in-person or virtually from wherever you are on November 9-10. We’ll see you there!

<p style="text-align:center;"><img src="https://user-images.githubusercontent.com/49915749/195142431-00e05566-e0ec-4b80-a83c-824ee7d38df9.png" alt="Logo"></p>

**Comment by ajay-dhangar** [comment]
![image](https://user-images.githubusercontent.com/99037494/195162082-120bc837-4f89-42a3-9728-7cb1c49d2caf.png)
**Go to and register now**

**Reply by jerryldh** [reply]
###register？

**Reply by joomoraes** [reply]
Nice

**Comment by anaclumos** [comment]
For those who are looking for a direct register link: [Register GitHub Universe 2022](https://register.githubuniverse.com/)

**Comment by larrnahmad** [comment]
> ![image](https://user-images.githubusercontent.com/99037494/195162082-120bc837-4f89-42a3-9728-7cb1c49d2caf.png)
> **Go to and register now**

**Reply by uwussimo** [reply]
nice wallpaper btw!

**Comment by ThatOneCalculator** [comment]
$320 for a one day ticket??? Why not make it at least somewhat affordable, similar events are ~$100ish...

**Reply by Fatsek-tuvpo9-vawcem** [reply]
918310

**Comment by Yunnathai** [comment]
i love you guys . if you watch this coment.

**Reply by mohammadwajahat** [reply]
https://e-commercepk.com/

**Comment by KPI0** [comment]
ok

**Comment by moshfiqrony** [comment]
Done

**Reply by ThatOneCalculator** [reply]
How about you get me a ticket too while you're at it 😇

**Comment by thebulbulbhan** [comment]
Excited

**Comment by MitkoVtori** [comment]
I registered, also is there gonna be a badge for it?

**Comment by bobrov-site** [comment]
❤️👨‍💻

**Comment by musunny4742** [comment]
Very Nice 🙂

**Comment by uwussimo** [comment]
nice wallpaper btw!

**Comment by mohammadwajahat** [comment]
https://e-commercepk.com/

**Comment by Glitch143** [comment]
Hello

**Comment by amw514** [comment]
What's this event? What's its objectives?

**Reply by ghost** [reply]
Checkout: https://githubuniverse.com/events/detail/virtual-schedule

**Reply by amw514** [reply]
thanks

**Comment by balookhan67** [comment]
Wow

**Comment by Naviruiz** [comment]
👍🇲🇽😎

**Comment by farzan-dehbashi** [comment]
Great!

**Comment by zafeirisdimi** [comment]
💺 I have already registered for **GitHub universe 2022**. See you all there.

**Comment by gaderz** [comment]
I'm currently new here , still trying to figure out how this works and how I can host my codes here

**Reply by Whiskey992** [reply]
It's for idea purposes and forking but depending on your code.

**Comment by sombewendin** [comment]
Hello . I'm also new here . I don't even know how to host my codes here

**Comment by gibbiemonster** [comment]
Can't wait to see you all!

**Comment by LightxBearer84** [comment]
How do I fix a problem on my app that's been coded

**Comment by y4my4my4m** [comment]
This comment section is a nightmare and those tickets are overpriced.
So, I'm just gonna shill my own thing until github realizes the comments on frontpage announcements were a bad idea

Check it out, shader wallpapers working on KDE, you can even load them from shadertoy.com
https://github.com/y4my4my4m/kde-shader-wallpaper
![image](https://user-images.githubusercontent.com/8145020/195466191-e9c51f5e-b141-43ab-b670-eaece50ee8e8.png)

**Reply by virtualWinter** [reply]
nice but why manjaro

**Comment by mike2339** [comment]
Interested,can't wait to see the new features ❤️

**Comment by dolorain** [comment]
its cool.

**Comment by LucasEduardoLins** [comment]
Good. ❤️

**Comment by Whiskey992** [comment]
Who's Satoshi

**Reply by ultrasoundcode** [reply]
It's me 🙃

**Reply by Wonnythatman** [reply]
WonnyThatman https://g.co/kgs/m8c8Yf

**Comment by appstarbd** [comment]
Excited

**Comment by hal0n0sec** [comment]
👍

**Comment by JeongSooNa** [comment]
I already registered! But Virtual Experience..
I'm very interesting because of my first universe in currier.
I need more help for understand Github universe.
Because of my english, development skill and experience.
Thank U 😄

**Comment by Deali-Axy** [comment]
Nice~😝

**Comment by aniskhan001** [comment]
Why not allow us to log in via GitHub account directly for registration? 🙂

**Reply by Whiskey992** [reply]
Github is based for business ideas and collaboration from community to see if your business had a spot in the real world not to verify

**Comment by meliy-meyada** [comment]
Excited😱

**Comment by Jamez-Eddy** [comment]
🤤💖

**Comment by Chrono-x55** [comment]
> Cool stuff

**Comment by playcreate228** [comment]
Hello

**Comment by playcreate228** [comment]
Hello

**Comment by marolansambi** [comment]
That's right 👍

**Comment by men-tor-hope** [comment]
I'm a novice here. Can someone please help me understand the platform better

**Reply by dl532** [reply]
https://docs.github.com/en/get-started/quickstart/hello-world

**Comment by Rakesh11sk** [comment]
Nice

**Comment by ctkqiang** [comment]
Noice👍🏼

**Comment by WalkAlone0325** [comment]
Nice.

**Comment by Code-SHD** [comment]
Wow

**Reply by fastmicro** [reply]
booya

**Comment by QuiTran86** [comment]
Interested!

**Reply by Wonnythatman** [reply]
WonnyThatman https://g.co/kgs/m8c8Yf

**Comment by davinx69** [comment]
Hello here any content to work with?

**Reply by Wonnythatman** [reply]
WonnyThatman https://g.co/kgs/m8c8Yf

**Comment by Esthemes** [comment]
"DITTO" you know the whole new and unfamiliar thing. Can't wait to learn to navigate my way around. More than anything can't wait to both learn and show what I know

**Reply by Wonnythatman** [reply]
WonnyThatman https://g.co/kgs/m8c8Yf

**Comment by Wonnythatman** [comment]
WonnyThatman https://g.co/kgs/m8c8Yf

**Comment by Wonnythatman** [comment]
WonnyThatman https://g.co/kgs/m8c8Yf

**Comment by NitinSN711** [comment]
Hi Guys......i am new here can you help me how to host my codes here

**Comment by troysjanda** [comment]
Doing the Virtual attendance.

**Comment by timblac** [comment]
Nice

**Comment by MuriloEduardo** [comment]
I want, please!

**Comment by tekeyew97** [comment]
How are you guys

**Comment by Kalutu** [comment]
Eagerly waiting

**Comment by tekeyew97** [comment]
Украина киев

**Comment by tekeyew97** [comment]
Киев

**Comment by novium2** [comment]
Retiring this post as it is over 6 months old. If you have any questions, consider starting a [new post](https://github.com/orgs/community/discussions/new/choose).
